control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -449,7 +449,6 @@ func ReconcileServiceMonitor(sm *prometheusoperatorv1.ServiceMonitor, ownerRef c
 	targetPort := intstr.FromString("https")
 	sm.Spec.Endpoints = []prometheusoperatorv1.Endpoint{
 		{
-			Interval:   "15s",
```

Comment:
what's default now that's not set?

Reply:
whatever is configured in the prometheus doing the scraping

Reply:
As in https://github.com/prometheus/prometheus/blob/main/documentation/examples/prometheus.yml#L3